release/changes.md

```diff
@@ -0,0 +1,2 @@
+- [NEW] - Run tests against AGP 8.1.0-beta02
```

Comment:
I filed a ticket to improve this: https://github.com/gradle/android-cache-fix-gradle-plugin/issues/508